packages/core/sample-projects/Development.md

```diff
@@ -0,0 +1,25 @@
+# Development
+
+## Environment
+
+You need to start a redis service. And the redis's default port is 6379. You can edit it at `kuai.config.ts`
```

Comment:
`Subject` in sentences should be the service/program instead of `human`.
`You need`, `You should` sound like commands, which is not polite.

In some cases, `I think`, `I figure` sound not objective.